### ISSUE TITLE
ImportError: No module named builtins

### ISSUE THREAD

**Issue by chaverman-trulia** [dpranke/pyjson5#18]
Recently upgraded to 0.5 in my app. Ran it and got an `ImportError`:

```
Traceback (most recent call last):
...
  File "/path/to/app/api.py", line 4, in <module>
    from json5 import loads
  File "/path/to/virtualenv/local/lib/python2.7/site-packages/json5/__init__.py", line 17, in <module>
    from . import tool
  File "/path/to/virtualenv/local/lib/python2.7/site-packages/json5/tool.py", line 28, in <module>
    from . import lib
  File "/path/to/virtualenv/local/lib/python2.7/site-packages/json5/lib.py", line 19, in <module>
    from builtins import str
ImportError: No module named builtins
```

This is running under Python 2.7.6.

I suspect the error happens because `builtins` is in the standard library in Python 3 but does not exist in Python 2. I resolved it by installing the `future` package from PyPI. As for the fix in this project, I'm not sure if including `future` works flawlessly when operating under Python 3.

**Comment by dpranke** [comment]
Odd, it looks like you are right, though I don't think I ever explicitly installed a `builtins` package in my 2.7 install. I will fix this ASAP.

**Comment by dpranke** [comment]
This should be fixed in b2a0304. I will release a v0.5.1 hopefully later today after I've done a bit more testing.

**Comment by dpranke** [comment]
I've pushed v0.5.1 to PyPI now.
